### PR TITLE
[release-ocm-2.4] MGMT-9102: Change the base image the same as in master

### PR DESCRIPTION
**Pull request by eliorerz**
Backport of:
 - https://github.com/openshift/assisted-test-infra/pull/1255
- https://github.com/openshift/assisted-test-infra/pull/1410
- https://github.com/openshift/assisted-test-infra/pull/1258
- https://github.com/openshift/assisted-test-infra/pull/1348
- https://github.com/openshift/assisted-test-infra/pull/1173 
- https://github.com/openshift/assisted-test-infra/pull/1269
- https://github.com/openshift/assisted-test-infra/pull/1198
- https://github.com/openshift/assisted-test-infra/pull/1372

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -162,7 +162,7 @@ create_full_environment:
 create_environment: image_build bring_assisted_service start_minikube
 
 image_build:
-	sed 's/^FROM .*assisted-service.*:latest/FROM $(subst /,\/,${SERVICE})/' Dockerfile.test-infra | \
+	sed 's/^FROM .*assisted-service.*:latest/FROM $(subst /,\/,${SERVICE})/' Dockerfile.assisted-test-infra | \
```
Comment (osherdp) [inline Makefile:165]: maybe we can give up on renaming the dockerfile

File: Dockerfile.assisted-test-infra [hunk 2]
```diff
@@ -18,15 +18,17 @@ RUN yum -y install \
...
 
 RUN curl -Lo terraform.zip https://releases.hashicorp.com/terraform/0.14.11/terraform_0.14.11_linux_amd64.zip && unzip terraform.zip -d /usr/bin/ && rm -rf terraform.zip
 
-COPY requirements.txt /tmp/
+COPY requirements-39.txt requirements-dev.txt ./
 COPY --from=service /clients/assisted-service-client-*.tar.gz /build/pip/
 RUN pip3 install --upgrade pip && \
-      pip3 install --no-cache-dir -I -r /tmp/requirements.txt && \
+      pip3 install --no-cache-dir -I -r ./requirements-39.txt -r ./requirements-dev.txt && \
       pip3 install --upgrade /build/pip/*
```
Comment (osherdp) [inline Dockerfile.assisted-test-infra:29]: revert those changes

File: .ci-operator.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+build_root_image:
```
Comment (osherdp) [inline .ci-operator.yaml:1]: this should be accompanied with a change in release repo:
```
build_root:
  from_repository: true
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (eliorerz) [reply Makefile:165]: Already did on https://github.com/openshift/release/pull/25855

File: .ci-operator.yaml (hunk 3 above)
Comment (eliorerz) [reply .ci-operator.yaml:1]: Done on https://github.com/openshift/release/pull/25855

**Review by osherdp — APPROVED** [review]

**Comment by eliorerz** [comment]
/override ci/prow/e2e-metal-single-node-live-iso
/override  ci/prow/e2e-metal-assisted-none-ipv6 
/override  ci/prow/system-test-operator 
/override ci/prow/e2e-metal-assisted-kube-api-late-binding-single-node
/override  ci/prow/e2e-metal-assisted-olm

**Pull request merged**